targets/TARGET_Cypress/TARGET_PSOC6/TARGET_CY8CKIT_062_WIFI_BT_PSA/prebuilt/README.md

```diff
@@ -0,0 +1,23 @@
+
```

Comment:
Make this relevant to your target please

Reply:
Done.

Reply:
@orenc17 I see that you have updated the Readme.md in the Future_Sequana target a week ago. But I have used as a template for our target the previous version of the file.
Is it better to change our file accoording to the current Future_Sequana version?
By the way - the command that is provided in the Future Readme.md for psa tests does not work for me. There are failures on psa_*_tests_part1_partition.c files compilation. How can it be fixed?